lib/tasks/sam.rake

```diff
@@ -1,9 +1,9 @@
 namespace :sam do
   task check: :environment do
-    User.not_in_sam.each do |user|
+    User.where(sam_account: false).each do |user|
```

Comment:
Does it make sense to have a class method like `SamAccountReckoner.unreconciled` that wraps this `where` clause. That way the Rake task doesn't have to know anything about the schema structure of the User table